src/ui/page/share/dataset/steps/dataSelection/dataSelection.tsx

```diff
@@ -17,6 +28,21 @@ export const DataSelection: FC = () => {
     removeFile: state.removeFile,
     storeFiles: state.storeFiles
   }))
+  const dispatchNotification = useDispatchNotification()
+
+  const handleFileError = useCallback(
+    (error: FileError) => {
+      const notificationInput: DispatchNotificationInput = {
+        type: 'error',
+        titleKey: 'error.problem',
+        messageKey: 'error.processing',
+        action: 'refresh'
+      }
+      console.error(ShowFileError.show(error))
```

Comment:
You can wrap the error message here to bring more context if you want, or to make the logged message more friendly.

Reply:
@ccamel 
AFAIS, the error messages returned by the `show` function look pretty standalone as they contain the whole context related to the use-case.

So, I'm not sure here we need to provide additional context, so, if you are OK, I'll suggest to keep it like it is.